test/js/gc-fn.js

```diff
@@ -0,0 +1,10 @@
+if (global.gc) {
+  module.exports = global.gc
+  return
```

Comment:
Tiniest of slightest of micro-nits: can you add semicolons here?

Reply:
Sure sorry, I was working in a repo without semicolons when I wrote this originally 😆